action.yaml

```diff
@@ -156,33 +156,42 @@ runs:
           curl -vvvX POST '${{ env.CITADEL_URL }}' --silent --fail --show-error -d @$file_path -H 'Content-Type: application/json' -H "Connection: close"  
           sleep ${{ env.CITADEL_JSON_LIST_SLEEP_SECONDS || 60  }}   # If env not passed, Sleeping for 60 seconds (default)
         done
-        
-    # Blackduck section
-    - name: Blackduck Scan
-      if: ${{ env.BlackDuck_Project_Name }}
-      id: blackduck-scan
-      uses: addnab/docker-run-action@v3
-      with:
-        image: docker.repo.orl.eng.hitachivantara.com/blackducksoftware/detect:8
-        options: --entrypoint "/bin/bash" -v ${{ github.workspace }}:/workdir
-        run: |
-          java -jar /synopsys-detect.jar \
-           --detect.source.path=${{ env.BlackDuck_Source_Path || '/workdir' }} \
-           --detect.project.version.name="${{ env.BlackDuck_Project_Version }}" \
-           --detect.project.name="${{ env.BlackDuck_Project_Name }}" \
-           --detect.accuracy.required=NONE \
-           --blackduck.api.token="${{ env.BlackDuck_Api_Token }}" \
-           --blackduck.url="${{ env.BlackDuck_Url }}" \
-           --detect.blackduck.signature.scanner.snippet.matching="NONE" ${{ env.ADDITIONAL_ARGS }}
-
-    - name: Blackduck scan tracking
+
+    - name: Citadel scan tracking    
       shell: bash
-      if: ${{ always() && env.BlackDuck_Project_Name }}
+      if: ${{ always() && env.TRIGGER_CITADEL_SCAN }}
       run: |
         icon="boom"
-        [[ "${{ steps.blackduck-scan.outcome }}" == "success" ]] && icon="white_check_mark"
+        [[ "${{ steps.citadel-scan.outcome }}" == "success" ]] && icon="white_check_mark"
 
-        sed -i "s/blackduck_scan=x/blackduck_scan=${icon}/g" $RUNNER_TEMP/icons.properties
+        sed -i "s/citadel_scan=x/citadel_scan=${icon}/g" $RUNNER_TEMP/icons.properties
```

Comment:
I don't think we want to remove the Black duck step completely.

Reply:
Ok. For now added Blackduck as commented code

Reply:
But why? If it's not being called, doesn't matter if the code is there. The OWASP step code is also there even if it's not being called.

Reply:
yes we keep that code. But for tracking do we still need to track blackduck along with Citadel ??

Reply:
For what we know at the moment, we only track Citadel.

Reply:
OK

Reply:
Uncommented the blackduck step